massa-models/src/output_event.rs

```diff
@@ -0,0 +1,44 @@
+use serde::{Deserialize, Serialize};
+
+use crate::{Address, BlockId, DeserializeCompact, SerializeCompact, Slot};
+
+#[derive(Debug, Clone, Serialize, Deserialize)]
+/// By product of a byte code execution
+pub struct SCOutputEvent {
+    context: ExecutionContext,
+    /// json data string
+    data: String,
+}
+
+#[derive(Debug, Clone, Serialize, Deserialize)]
+pub struct ExecutionContext {
+    slot: Slot,
+    bytes_address: Address,
+    caller_address: Address,
+    block: Option<BlockId>,
+    // todo I don't know how to represent the call stack
```

Comment:
I think this has already been done [here](https://github.com/massalabs/massa/blob/399fb6e02a977903d3e085084978c671b19403ba/massa-execution/src/types.rs#L31)? cc @damip

Reply:
yes indeed